R/random-forest-deployment.R

```diff
@@ -7,24 +7,43 @@
 #' \item Push these predictions to SQL Server
 #' }
 #' @docType class
-#' @usage RandomForestDeployment(type, df, grainCol, testWindowCol, 
-#' predictedCol, impute, debug)
+#' @usage RandomForestDeployment(type, df, grainCol, predictedCol, impute, debug, cores, modelName)
```

Comment:
is "trees" not a modifiable param in rf deploy? If it actually is, it should probably be listed in the params and in the example.